homeassistant/components/homeassistant/triggers/time.py

```diff
@@ -45,16 +47,28 @@
     }
 )
 
+
+def valid_at_template(value: Any) -> template.Template:
+    """Validate either a jinja2 template, valid time, or valid trigger entity."""
+    tpl = cv.template(value)
+
+    if tpl.is_static:
+        _TIME_AT_SCHEMA(value)
+
+    return tpl
+
+
 _TIME_TRIGGER_SCHEMA = vol.Any(
     cv.time,
     _TIME_TRIGGER_ENTITY,
     _TIME_TRIGGER_ENTITY_WITH_OFFSET,
+    valid_at_template,
     msg=(
-        "Expected HH:MM, HH:MM:SS, an Entity ID with domain 'input_datetime' or "
-        "'sensor', or a combination of a timestamp sensor entity and an offset."
+        "Expected HH:MM, HH:MM:SS, Entity ID with domain 'input_datetime' or 'sensor', or Limited Template"
```

Comment:
Was the changed text from PR #120858 lost in rebase?

Reply:
Probably, I'll fix this.